src/tools/python/python.ts

```diff
@@ -158,14 +175,7 @@ export class PythonTool extends Tool<PythonToolOutput, PythonToolOptions> {
     _options: BaseToolRunOptions | undefined,
     run: RunContext<this>,
   ) {
-    const inputFiles = await this.storage.upload(
-      Object.entries(input.inputFiles ?? {})
-        .filter(([k, v]) => Boolean(k && v))
-        .map(([id, filename]) => ({
-          id,
-          filename: filename!.split(":").at(-1) ?? filename!,
-        })),
-    );
+    const inputFiles = await this.storage.upload((input.inputFiles as PythonFile[]) ?? []);
```

Comment:
This changes the behavior. In the prior implementation, we filtered out `falsy` values.

Reply:
That's redundant, it's not allowed by the JSON schema.